benchmarks/src/main/scala/io/finch/benchmarks.scala

```diff
@@ -181,6 +182,39 @@ class ToServiceBenchmark extends FinchBenchmark {
   def ints: Response = Await.result(intService(Request()))
 }
 
+@State(Scope.Benchmark)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+class BootstrapBenchmark extends FinchBenchmark {
```

Comment:
I'd probably vote to unify our ToService and Boostrap benchmarks, but I'm happy to do that in a separate PR.